util/wait-for-app.sh

```diff
@@ -0,0 +1,19 @@
+#!/bin/bash
+
+if [ -z $1 ]
+then
+  echo "No URL specified, using http://localhost:8080/"
+  APP_URL='http://localhost:8080/'
+else
+  APP_URL=$1
+fi
+
+# wait for app to start up
+echo "Waiting for application startup"
+until curl -s $APP_URL > /dev/null
+do
+  echo -n "." # waiting
+  sleep 2
```

Comment:
perhaps a timeout here would be good? e.g. maximum 3 minutes?

Reply:
Good point, will add it!